common/app/views/fragments/footer.scala.html

```diff
@@ -151,17 +151,7 @@
             data-link-name="footer : contribute-cta"
             data-edition="@{editionId}"
             href="@getReaderRevenueUrl(SupportContribute, Footer)">
```

Comment:
Does this URL need to be updated at all?

Reply:
No, the spec just calls for a single CTA, whose label says 'Support', and which points to the normal contributions landing page.